docusaurus/docs/user-docs/intro.md

```diff
@@ -9,115 +9,46 @@ sidebar_label: Welcome!
 
 The documentation for Strapi contains 3 main sections, accessible from the top navigation bar:
 
-- 🧑‍💻 The **[Developer Docs](/dev-docs/intro)**  all the technical information related to the setup, advanced usage, customization, and update of your Strapi v5 application.
-- 🧑‍🏫 The **User Guide**  that you're currently reading contains all about using Strapi's admin panel.
+- 🧑‍💻 The **[Developer Docs](/dev-docs/intro)** contain all the technical information related to the setup, advanced usage, customization, and update of your Strapi v5 application.
```

Comment:
```suggestion
- 🧑‍💻 The **[Developer Docs](/dev-docs/intro)** contain all the technical information related to the setup, advanced usage, customization, and update of your Strapi v4 application.
```

This was a typo I had in the Dev Docs too 🤦‍♂️ 😄 I've just [fixed it](https://github.com/strapi/documentation/commit/6caf6592824a50032f2e15a2f5a15657409c69bd) in the Dev Docs, could you please fix it here too?

Reply:
Okay, I've done that thank you.